internal/tsc_wrapped/compiler_host.ts

```diff
@@ -256,12 +275,13 @@ export class CompilerHost implements ts.CompilerHost, tsickle.TsickleHost {
       onError?: (message: string) => void,
       sourceFiles?: ts.SourceFile[]): void {
     // Workaround https://github.com/Microsoft/TypeScript/issues/18648
-    if (fileName.endsWith('.d.ts') && sourceFiles && sourceFiles.length > 0 &&
-        sourceFiles[0].moduleName &&
-        (this.options.module === ts.ModuleKind.AMD ||
-         this.options.module === ts.ModuleKind.UMD)) {
-      content =
-          `/// <amd-module name="${sourceFiles[0].moduleName}" />\n${content}`;
+    if ((this.options.module === ts.ModuleKind.AMD ||
```

Comment:
just changing the order here as a micro-optimization: in google3 we should short-circuit at the first condition